src/v2/views/board/NewContactGroupBoard.tsx

```diff
@@ -0,0 +1,352 @@
+
+	const onTitleChange = useCallback<NonNullable<InputProps['onChange']>>(
+		(ev) => {
+			setTitleValue(ev.target.value);
+			if (size(ev.target.value) === 0) {
+				updateBoard({ title: '' });
+			} else {
+				updateBoard({ title: ev.target.value });
+			}
+		},
+		[updateBoard]
+	);
+
+	const discardChanges = useCallback(() => {
+		setTitleValue(initialTitle);
+		setContactInputValue([]);
+		setMemberListEmails([]);
+		updateBoard({ title: initialTitle });
+	}, [initialTitle, updateBoard]);
+
+	const onSave = useCallback(() => {
+		client
+			.createContactGroup(titleValue, memberListEmails)
+			.then(() => {
+				createSnackbar({
+					key: new Date().toLocaleString(),
+					type: 'success',
+					label: t(
+						'board.newContactGroup.snackbar.contact_group_created',
+						'Contact group successfully created'
+					)
+				});
+				closeBoard();
+			})
+			.catch(() => {
+				createSnackbar({
+					key: new Date().toLocaleString(),
+					type: 'error',
+					label: t('label.error_try_again', 'Something went wrong, please try again')
+				});
+			});
+	}, [closeBoard, createSnackbar, memberListEmails, t, titleValue]);
+
+	const titleDescription = useMemo(() => {
+		if (titleValue.trim().length === 0) {
+			return t(
+				'board.newContactGroup.input.title_input.error.required',
+				'Group name is required, enter a name to proceed'
+			);
+		}
+		if (titleValue.length > CONTACT_GROUP_TITLE_MAX_LENGTH) {
+			return t(
+				'board.newContactGroup.input.title_input.error.max_length',
+				'Maximum length allowed is 256 characters'
+			);
+		}
+		return undefined;
+	}, [t, titleValue]);
+
+	const contactInputDescription = useMemo(() => {
+		let valid = 0;
+		let duplicated = 0;
+		let invalid = 0;
+
+		contactInputValue.forEach((value) => {
+			if (value.duplicated) {
+				duplicated += 1;
+			} else if (value.error) {
+				invalid += 1;
+			} else {
+				valid += 1;
+			}
+		});
+		if (valid > 0) {
+			return undefined;
+		}
+		if (invalid > 0 && duplicated > 0) {
+			return t(
+				'board.newContactGroup.input.contact_input.error.invalid_already_present_addresses',
+				'Invalid and already present addresses'
+			);
+		}
+		if (invalid > 0 && duplicated === 0) {
+			return t('board.newContactGroup.input.contact_input.error.invalid_address', {
+				count: invalid,
+				defaultValue: 'Invalid address',
+				defaultValue_plural: 'Invalid addresses'
+			});
+		}
+		if (duplicated === 1 && invalid === 0) {
+			return t(
+				'board.newContactGroup.input.contact_input.error.address_already_present',
+				'Address already present',
+				{
+					count: duplicated
+				}
+			);
+		}
+		if (duplicated > 1 && invalid === 0) {
+			return t(
+				'board.newContactGroup.input.contact_input.error.address_already_present',
+				'Addresses already present',
+				{
+					count: duplicated
+				}
+			);
+		}
+		return undefined;
+	}, [contactInputValue, t]);
+
+	const contactInputOnChange = (
+		newContactInputValue: Array<{
```

Comment:
Better if we use Array<ChipItem & { email: string; duplicated?: boolean; }>

Also it might be convenient to create a specific type that can be reused